src/sources/forus-platform/js/angular-1/services/ProductReservationsExportService.js

```diff
@@ -0,0 +1,70 @@
+const ProductReservationsExportService = function(
+    appConfigs,
+    FileService,
+    ModalService,
+    ProductReservationService,
+    PageLoadingBarService,
+    PushNotificationsService
+) {
+    return new (function() {
+        const dataFormats = [
+            { value: 'csv', label: 'Exporteren CSV', icon: 'file-delimited-outline' },
+            { value: 'xls', label: 'Exporteren XLS', icon: 'file-excel-outline' },
+        ];
+
+        const makeSections = (fields) => ([
+            { type: "radio", key: "data_format", fields: dataFormats, value: "csv", title: 'Kies bestandsformaat' },
+            { type: "checkbox", key: "fields", fields, fieldsPerRow: 3, selectAll: true, title: 'Kies inbegrepen velden' },
+        ]);
+
+        const saveExportedReservations = (data, organization_id, res) => {
+            PushNotificationsService.success('Success!', 'The downloading should start shortly.');
```

Comment:
needs translation and reformulate

Reply:
Success! = Succes!

The downloading should begin shortly. = De download begint over enkele ogenblikken.

(This is based on text already found in the code in VoucherExportService.js)